ejercicio1.rb

```diff
@@ -0,0 +1,54 @@
+require 'net/http'
+require 'uri'
+require 'json'
+require 'httparty'
+
+begin
+  puts "ingrese ruta dal archivo: "
+  namefile = gets.chomp
+  content = File.read(namefile)
+  puts "descripcion: "
+  description = gets.chomp
+  puts "Quieres el gist publico? si/no"
+  public = gets.chomp
+  
+  if public == 'si'
+    public = true
+  elsif public == 'no'
+    public = false
+  else
+    puts "respuesta incorrecta"
```

Comment:
Check what happens if the program enters here. What would be the value of "public"? Is the request made?